presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -175,6 +175,17 @@ Statistical Functions
     Returns the upper bound of the Wilson score interval of a Bernoulli trial process
     at a confidence specified by the z-score ``z``.
 
+.. function:: KLEINBERG_BURST_DETECTION(time[, s, gamma]) -> array<int>
+
+    Given an array of sorted timestamps, returns an array with the same length
+    as the input where a 0 value indicates no-burst, and any other value
+    indicates that the timestamp at this index corresponds to a burst whose
+    center is located in the index pointed by the value. The default value for
+    ``s`` is ``2.0`` and for ``gamma`` is ``1.0``::
+
+    SELECT KLEINBERG_BURST_DETECTION(ARRAY [100, 100.1, 101, 110, 110.1, 110.2, 110.3, 110.35, 110.40, 150, 304.35]); -- [0, 2, 2, 2, 6, 6, 6, 6, 6, 0, 0]
```

Comment:
Use lowercase for function names in documentation. See array functions for an example.